google/cloud/vision/entity.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Entity class for holding information returned from annotating an image."""
+
+
+from google.cloud.vision.geometry import Bounds
+
+
+class EntityAnnotation(object):
+    """Representation of an entity returned from the Vision API."""
```

Comment:
Added! I was cranking these out and I forgot to recheck them. I thought this would show up as an error when you run `tox -e docs`.